src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void getArea() {
+        double area = (double) Math.round(Math.PI * radius * radius);
+        System.out.println("Figure: circle, area: " + area + " sq.units, radius: "
```

Comment:
You need to create two interfaces: 
one with the `getArea()` method (you already have it), 
and another with the `draw()` method. 
Inside the `draw()` method, you should use `System.out.println` to display all the information about the figure.